src/Compilers/CSharp/Portable/Binder/Binder.QueryUnboundLambdaState.cs

```diff
@@ -26,6 +26,7 @@ public QueryUnboundLambdaState(Binder binder, RangeVariableMap rangeVariableMap,
                 _bodyFactory = bodyFactory;
             }
 
+            public override bool UnderscoreMeansDiscard { get { return false; } }
```

Comment:
`=> false`?

Reply:
I don't feel strongly, was trying to match surrounding code...

---
In reply to: [326826367](https://github.com/dotnet/roslyn/pull/38786#discussion_r326826367) [](ancestors = 326826367)